com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/cf/v1_0/CloudModelBuilderTest.java

```diff
@@ -332,8 +332,7 @@ public void setUp() throws Exception {
         DescriptorHandler handler = getDescriptorHandler();
 
         String targetName = extensionDescriptor.getDeployTargets().get(0);
-        Target implicitTarget = getDeployTargetFactory().create(targetName, platforms.get(0).getName());
-        Target target = handler.findTarget(targets, targetName, implicitTarget);
+        Target target = handler.findTarget(targets, targetName, null);
```

Comment:
Why is this changed to null? The DetectTargetStep still computes an implicit target during its execution.

Reply:
Because all targets are described in targets.json and in tests all targets are referenced from this file.
So this implicit target will never work because this create method requires org and space parameters.